openmmforcefields/generators/template_generators.py

```diff
@@ -594,11 +607,11 @@ def generator(self, forcefield, residue):
 
         """
         # Load the GAFF parameters if we haven't done so already for this force field
-        if forcefield not in self._gaff_parameters_loaded:
-            # Instruct the ForceField to load the GAFF parameters
-            forcefield.loadFile(self.gaff_xml_filename)
-            # Note that we've loaded the GAFF parameters
-            self._gaff_parameters_loaded[forcefield] = True
+        # if forcefield not in self._gaff_parameters_loaded:
+        #    # Instruct the ForceField to load the GAFF parameters
+        #    forcefield.loadFile(self.gaff_xml_filename)
+        #    # Note that we've loaded the GAFF parameters
+        #    self._gaff_parameters_loaded[forcefield] = True
```

Comment:
We don't need to subclass anymore since we don't load in the gaff xmls -- I will remove it but for testing I just commented it out for now